workspace.json

```diff
@@ -313,6 +313,16 @@
             ]
           }
         },
+        "init-schema": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "commands": [
+              {
+                "command": "yarn nx run gjafakort-api:codegen"
+              }
+            ]
```

Comment:
All of these `run-commands` can be simplified after you merge from master.

Reply:
Yes I saw that, hopefully, won't get too many conflicts 😬